### DISCUSSION TITLE
Can't reset character?

### DISCUSSION THREAD

**Discussion by cutesexyrobots** [drizzle-mizzle/CharacterAI-Discord-Bot#67] [Q&A] [answered]
First off, this is so cool and thank you for making it!
When I go to reset the character, no matter what way I do it, I get the "no permissions" gif. Everything else seems to work fine with the bot, I just can't reset them no matter what I do. I gave the bot administrator permissions when I made it so I don't know why it's rejecting the command.
I am using the "auto-setup" because I was getting the index range error at first, and auto-setup seemed to fix everything except the reset chat.

**Comment by nathanjo18** [comment]
I had this exact same problem in a server which I don't own.
What solved it was creating the role labeled as "Manage Character" and having it myself.

**Comment by drizzle-mizzle** [comment, marked as answer]
As @nathanjo18 said, you either have to be a server owner, or to have a bot role.
